### ISSUE TITLE
Suggest: Handle error messages from the server

### ISSUE THREAD

**Issue by mal-tee** [datenanfragen/website#1047]
I want the backend to be able to reject suggestions with a message. Before implementing this in the backend, I will implement support for it in the frontend.

This will allow us to filter stuff in the backend (e.g. white space changes) to reduce noise in the data repo, without confusing users.

**Comment by baltpeter** [comment]
We do have this already but I that's not what you're looking for? 

https://github.com/datenanfragen/website/blob/e98e8e5af5632ead32e53b591cad7d131ca8e33b/src/suggest-edit.js#L312-L319

**Comment by mal-tee** [comment]
Hmm, right. It's a bit generic and w/o a custom message, but should do the trick.

**Comment by baltpeter** [comment]
I mean, you could also add a way to set a custom error message for those. I'm just not quite sure what you want to implement yet. :D

**Comment by mal-tee** [comment]
> I'm just not quite sure what you want to implement yet. :D

E.g. I want the backend to check if value updates are just white space changes and omit them to reduce noise like this:

https://github.com/datenanfragen/data/pull/2134/files#diff-2e2c693eeb489acb22a617dd6fe60a96dd1f1db5b2c04418157418d52b5e49d5L16-R18

But this could result in edge cases where the server decides that a suggestion isn't post-worthy and therefore won't generate a PR. Since this is different than a straight up error I thought we might want to cover this via a message. 

But since you brought up the issue of translating these messages, I think we could just use the generic error. :D

I also have other ideas, e.g. withholding PRs that contain the string "gmail"... :D

**Comment by baltpeter** [comment]
> E.g. I want the backend to check if value updates are just white space changes and omit them to reduce noise like this:

Oh btw, I think you've suggested this before and I disagreed but I have now changed my mind on this. Let's force-format everything: https://github.com/datenanfragen/data/issues/2165, https://github.com/datenanfragen/data/issues/2166

> But since you brought up the issue of translating these messages, I think we could just use the generic error. :D

Wouldn't be too bad. You would just return a translation key from the server. *shrug*

> I also have other ideas, e.g. withholding PRs that contain the string "gmail"... :D

Oh, that's a good idea. But unfortunately:

https://github.com/datenanfragen/data/blob/4965796bb876a20788e131f1b3a7d896166f83ad/companies/tierschutzallianz.json#L12

You'd need a way to have the user confirm that this really is the company's email and not theirs.
